stdlib/SuiteSparse_jll/Project.toml

```diff
@@ -1,11 +0,0 @@
-name = "SuiteSparse_jll"
```

Comment:
I don't think we can just delete a stdlib like this.

Reply:
Ok - but I don't understand why not. Isn't it just a JLL, and the contents of it are provided by SuiteSparse_LBT_jll. Like I said, we can either update SuiteSparse_jll or use this one.

Reply:
I thought it was actually used, but turns out there are a bunch of stdlibs that are completely unused, xref https://github.com/JuliaLang/julia/pull/39127#discussion_r577701193

Reply:
Yeah, I am realizing that too now. None of the stdlibs declare dependencies on the relevant JLLs in their Project.toml, but perhaps they should.

Reply:
The JLLs are not currently used by Base Julia, but they will be in v1.7.  They're important to provide here in the event that a user package wants to directly interact with any of the libraries wrapped by the JLLs; e.g. if you want to use `libblastrampoline`, you need a way to get a handle on it, and that way is via the JLL.

Reply:
Why do they have to be stdlibs then? Seems like they could just be regular Julia packages if they are only used by regular Julia packages.